app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      redirect_to :root
+    else
+      flash.now[:danger] = t("invalid")
+      render "new"
```

Comment:
```suggestion
      flash.now[:danger] = t "invalid"
      render :new
```

tạo tên biến cho clear tí chứ `invalid` là gì, invalid cái j, chị đọc ko hiểu